.maintain/config/bridge-pangoro-kiln.toml

```diff
@@ -0,0 +1,10 @@
+[pangoro]
+endpoint = "https://pangoro-rpc.darwinia.network"
+contract_abi_path = "/path/to/BeaconLightClient_abi.json"
```

Comment:
Bundle contract to code?

Reply:
Emmm, Bundling abi into the code seems better, so useres don't need to find the abi by themselves. I'll update this.